### ISSUE TITLE
Can't compile pdf

### ISSUE THREAD

**Issue by happyrabbit** [happyrabbit/CE_JSM2017#3]
met all kinds of problem

**Comment by mingliisu** [comment]
Can we pull all .Rmd files to R Studio and then compile?

**Comment by mingliisu** [comment]
I am trying now, required to install a LaTex program

**Comment by happyrabbit** [comment]
When you compile pdf, it will generate a .rmd file that includes all the sections. If the pdf is compiled successfully, the new .rmd file will be deleted. Otherwise, it is there. You can work on the file in SubmitJuly folder. @mingliisu

**Comment by happyrabbit** [comment]
The error I got was:

pandoc: Error producing PDF
Error: pandoc document conversion failed with error 43

It is frustrating. God knows what is error 43...... *_*

**Comment by mingliisu** [comment]
Probably related to LaTex installation. I am trying to download a full version of MikeLaX to my computer and then try to compile it. I can successfully compile it to HTML without the nnet section as it requires to load data from your local path.

**Comment by happyrabbit** [comment]
Yes. I deleted nnet section. HTML always works.

**Comment by happyrabbit** [comment]
I reinstalled the updated latex which solved some problems but not all.

**Comment by mingliisu** [comment]
Looks like to docx version works too. I have download and opened it using Word, and we can save it as pdf :)

**Comment by mingliisu** [comment]
It is only 138 pages and I will do some editing/formatting tonight and save it as pdf file. No need to rush to solve the problem of using R Studio

**Comment by happyrabbit** [comment]
Yes, word works just fine. Is your computer Mac or Windows? It doesn't look nice in Mac. Are you working on the CE_JSM2017.rmd in SubmitJuly folder?

**Comment by happyrabbit** [comment]
Yes. Covert from word to pdf is a good idea!

**Comment by happyrabbit** [comment]
We should have no problem catching the July 6 deadline : )

**Comment by happyrabbit** [comment]
@mingliisu Ming，@yihui Told me there are many links start from /Users/happyrabbit/...... I need to change that.

**Comment by happyrabbit** [comment]
should be unhappy rabbit right now : (( @mingliisu @yihui

**Comment by mingliisu** [comment]
It is almost there! The draft handout in pdf and word are created

**Comment by happyrabbit** [comment]
从前，有一只快乐的兔子，用着bookdown，用着用着就得了忧郁症。 @yihui
